adaptor-rm-webapp/src/main/webapp/WEB-INF/jboss-all.xml

```diff
@@ -0,0 +1,3 @@
+<jboss xmlns="urn:jboss:1.0">
```

Comment:
Do we need to add these files, or should they be ignored?

Reply:
This one was also needed for WildFly server to disable CDI.